src/messages/manifestjson.js

```diff
@@ -143,6 +143,33 @@ export function manifestIconMissing(path) {
   };
 }
 
+export const MANIFEST_BACKGROUND_SCRIPT_NOT_FOUND = 'MANIFEST_BACKGROUND_SCRIPT_NOT_FOUND';
+export const MANIFEST_BACKGROUND_PAGE_NOT_FOUND = 'MANIFEST_BACKGROUND_PAGE_NOT_FOUND';
+export function manifestBackgroundMissing(path, type) {
+  if (type === 'script') {
+    return {
+      code: MANIFEST_BACKGROUND_SCRIPT_NOT_FOUND,
+      legacyCode: null,
+      message: _(
+        'A background script defined in the manifest could not be found.'),
+      description: sprintf(
+        _('Background script could not be found at "%(path)s".'),
+        { path }),
+      file: MANIFEST_JSON,
+    };
+  }
+  return {
+    code: MANIFEST_BACKGROUND_PAGE_NOT_FOUND,
+    legacyCode: null,
+    message: _(
+      'A background page defined in the manifest could not be found.'),
+    description: sprintf(
+      _('Background page could not be found at "%(path)s".'),
+      { path }),
+    file: MANIFEST_JSON,
+  };
+}
+
```

Comment:
Both return values are mostly identical, I'm sure you can unify that to one constant and just replace the `code`, `message` and `description`. Maybe, if not I'm fine with using it like that too